conda_forge_tick/deploy.py

```diff
@@ -93,10 +94,11 @@ def _pull_changes(batch):
     return n_added
 
 
-def _deploy_batch(*, files_to_add, batch, n_added, max_per_batch=200):
+def _deploy_batch(*, files_to_add: Iterable[str], batch, n_added, max_per_batch=200):
```

Comment:
```suggestion
def _deploy_batch(*, files_to_add: set[str] | list[str], batch, n_added, max_per_batch=200):
```

I think this will let us remove the cast below which would convert a set to a list when not needed.

Reply:
Python sets have a pop method: https://docs.python.org/3/library/stdtypes.html#frozenset.pop